src/app/dashboard/Missions/CreateMission.js

```diff
@@ -0,0 +1,482 @@
+            </MenuItem>
+          </Select>
+        </FormControl>
+        {missionType !== null ? (
+          <>
+            <Typography className={classes.inputHeader}>Select Food Box Options</Typography>
+            <FormControl className={classes.form}>
+              {Object.keys(foodBoxArray).map((boxNumber) => {
+                const { amount, name } = foodBoxArray[boxNumber];
+                return (
+                  <div className={classes.sideBySideSelect} key={boxNumber}>
+                    <Select
+                      onChange={handleItems}
+                      name={"item for box " + boxNumber}
+                      variant="outlined"
+                      className={classes.select}
+                      value={name}
+                    >
+                      <MenuItem value="Fruits and Veggies">Fruits & Veggies</MenuItem>
+                      <MenuItem value="Non-Perishables">Non-Perishables</MenuItem>
+                      <MenuItem value="Meat">Meat</MenuItem>
+                    </Select>
+                    <TextField
+                      onChange={handleItems}
+                      name={"amount for box " + boxNumber}
+                      type="number"
+                      variant="outlined"
+                      InputProps={{ inputProps: { min: 0 } }}
+                      className={classes.numSelector}
+                      value={amount}
+                    />
+                  </div>
+                );
+              })}
+              <Button className={classes.selectCreateButton} onClick={(e) => addToFBCount(e)}>
+                <AddCircleIcon style={{ marginRight: "5px", color: "#3739B5" }} />
+                Add Food Box
+              </Button>
+            </FormControl>
+          </>
+        ) : null}
+        <Divider variant="middle" />
+        <div className={classes.optionalTagBox}>
+          <Typography className={classes.inputHeader}>Funding Details </Typography>
+          <Typography className={classes.optionalTag}> - optional</Typography>
+        </div>
+        <FormControl className={classes.form}>
+          <Select
+            variant="outlined"
+            className={classes.select}
+            value={missionFund}
+            onChange={handleChangeFund}
+            startAdornment={
+              <InputAdornment position="start">
+                <AttachMoneyIcon style={{ color: "#3739B5" }} />
+              </InputAdornment>
+            }
+          >
+            {/* THIS WILL HAVE TO HAVE A DYNAMIC INITIAL VALUE */}
+            <MenuItem className={classes.menuItem} value="Not yet funded">
+              Not yet funded
+            </MenuItem>
+            <MenuItem className={classes.menuItem} value="Funded">
+              Funded
+            </MenuItem>
+          </Select>
+        </FormControl>
+        <Divider variant="middle" />
+        <div className={classes.dualForm}>
+          <FormControl className={classes.form2}>
```

Comment:
@thirdeyeclub Can you name the style classes with something more meaningful than "form2", eg. "recipientDetails"?